cg/services/delivery_message/utils.py

```diff
@@ -60,6 +64,31 @@ def get_message_strategy_from_data_delivery(case: Case) -> DeliveryMessage:
 }
 
 
+def get_rna_message_strategy_from_data_delivery(case: Case, store: Store) -> DeliveryMessage:
+
+    if case.data_delivery in scout_data_delivery:
+        return rna_message_map[case.data_delivery](store)
+    return rna_message_map[case.data_delivery]()
+
+
+scout_data_delivery: list[DataDelivery] = [
```

Comment:
It would be ice to define all these constant maps at the top of this file, just below the imports. This includes the map not added in this PR, just to be consistent. Also, lets name them with capital letters so it is clear that is a constant dictionary

Reply:
However, these maps are used in the functions directly above them and maybe it is easier to reads if they are closed to where they are used (and not used anywhere else). What do you think?